lib/Pipelines/ArithmeticPipelineRegistration.h

```diff
@@ -43,6 +43,10 @@ struct MlirToRLWEPipelineOptions
       llvm::cl::desc("Modulus switching right before the first multiplication "
                      "(default to false)"),
       llvm::cl::init(false)};
+  PassOptions::Option<bool> debug{
+      *this, "debug",
+      llvm::cl::desc("Enable debug mode for the pipeline (default to false)"),
+      llvm::cl::init(false)};
```

Comment:
The name should be more descriptive, since it collides (mentally) with the usual `--debug` option for MLIR.

What about 

```suggestion
  PassOptions::Option<bool> debug{
      *this, "insert-debug-handler-calls",
      llvm::cl::desc("Insert function calls to an externally-defined debug function (cf. --add-debug-port)"),
      llvm::cl::init(false)};
```